addons/binding/org.openhab.binding.zoneminder/build.properties

```diff
@@ -0,0 +1,7 @@
+source.. = src/main/java/
+output.. = target/classes
+bin.includes = META-INF/,\
+               .,\
+               OSGI-INF/,\
+               ESH-INF/, \
+               lib/zoneminder4j-0.1-jar-with-dependencies.jar
```

Comment:
Note that you should also add an about.html file, which names 3rd party libs that are included, with their location and license. See this example: https://github.com/openhab/openhab2-addons/blob/master/addons/binding/org.openhab.binding.chromecast/about.html